executor/aggregate.go

```diff
@@ -189,6 +189,9 @@ type HashAggExec struct {
 	prepared         bool
 	executed         bool
 
+	isAllFirstRow     bool
```

Comment:
Add a comment for `isAllFirstRow` and `firstRowProcessed`

Reply:
Done